debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/buffered/BufferedLogMinerStreamingChangeEventSource.java

```diff
@@ -805,6 +805,12 @@ private boolean isTransactionSkippedAtCommit(Transaction transaction) {
                     LOGGER.debug("Skipped transaction with excluded username {}", transaction.getUserName());
                     return true;
                 }
+                else if (!getConfig().getLogMiningUsernameIncludes().isEmpty()) {
```

Comment:
Do we need here and below nested conditions? I beleive we can either comapre directly as empty string will not match current username or the conditons can be put in a single statement with and operator.

The same applies to the change below.

Reply:
Hi @jpechane fixed, and I also chose to pull up the logic across both processors w/test adjustments.